components/sst_importer/src/sst_importer.rs

```diff
@@ -724,20 +752,29 @@ impl ImportFile {
 
     pub fn finish(&mut self) -> Result<()> {
         self.validate()?;
-        self.file.take().unwrap().sync_all()?;
+        self.file.take().unwrap().flush()?;
```

Comment:
Can you post the reference to illustrate `flush` has same semantic with `sync_all`